### ISSUE TITLE
PyPi dj-database-url points to different/old github codebase.

### ISSUE THREAD

**Issue by hcpatel** [jazzband/dj-database-url#153]
I have just realised that the PyPi dj-database-url is pointing to a forked version of this under a different name (https://github.com/kennethreitz/dj-database-url , 0.5.0 from 2018, and no recent activity), but this repo looks like it has more recnt active than that and more up to date with new Django versions.

Are you aware of this?

I am interested in using this. Hense, is there a 'stable/production' version of this available via Pip install?

Will your next release replace the above or be under a different name?

Kind Regards,
Hitesh Patel

**Comment by mattseymour** [comment]
@jacobian should be able to do this, or once the project is fully migrated to Jazzband then we can make the change.

**Comment by simkimsia** [comment]
my workaround is to use

```
git+https://github.com/jazzband/dj-database-url@v1.0.0#egg=dj-database-url
```

**Comment by edmorley** [comment]
Please can a new version of dj-database-url be released to PyPI?

The current version is still the 0.x release.

**Comment by mattseymour** [comment]
@edmorley the release process has been kicked off.

**Comment by mattseymour** [comment]
Marking as closed 1.0.0 release is live.
